src/docs/i18n/vi/docusaurus-plugin-content-docs/current/overview.md

```diff
@@ -4,117 +4,90 @@ sidebar_position: 1
+Để hỗ trợ tốt hơn việc khám phá các blog trong cộng đồng của chúng tôi, chúng tôi thiết lập blog nguồn mở [Planet](<https://en.wikipedia.org/wiki/Planet_(software)>): một nguồn cấp dữ liệu tổng hợp các bài đăng trên blog từ các giảng viên và sinh viên Seneca làm việc trên nguồn mở trong một trang duy nhất. Blog Planet của chúng tôi hiện đang hoạt động tại [http://zenit.senecac.on.ca/~chris.tyler/planet/](http://zenit.senecac.on.ca/~chris.tyler/planet/).
 
-## What is a Planet?
+## Planet là gì?
 
-> Planet is a feed aggregator application designed to collect posts from the weblogs of members of an Internet community and display them on a single page. Planet runs on a web server. It creates pages with entries from the original feeds in chronological order, most recent entries first. --[Wikipedia](<https://en.wikipedia.org/wiki/Planet_(software)>)
+> Planet là một ứng dụng tổng hợp nguồn cấp dữ liệu được thiết kế để thu thập các bài đăng từ nhật ký web của các thành viên trong cộng đồng Internet và hiển thị chúng trên một trang duy nhất. Planet chạy trên một máy chủ web. Nó tạo ra các trang và lấy dữ liệu từ nguồn cấp dữ liệu gốc theo thứ tự thời gian, các dữ liệu được nhập gần đây nhất trước. --[Wikipedia](<https://en.wikipedia.org/wiki/Planet_(software)>)
 
-In the early 2000s, before the rise of social media apps like Twitter and Facebook,
-Planet solved a important problem in the free and open source community. It used
-various "feed" technologies (RSS, Atom, CDF) to allow blog posts from different
-platforms to be aggregated into a single page that was constantly updated with
-the latest posts by people within a particular community.
+Vào đầu những năm 2000, trước sự trỗi dậy của các ứng dụng truyền thông xã hội như Twitter và Facebook, Planet đã giải quyết được một vấn đề quan trọng trong cộng đồng tự do và open source. Nó sử dụng các công nghệ “nguồn cấp dữ liệu” khác nhau như là RSS, Atom, CDF để cho phép các bài đăng trên blog từ các nền tảng khác nhau được tổng hợp thành một trang duy nhất, và được cập nhật liên tục với các bài đăng mới nhất của những mọi người trong một cộng đồng cụ thể.
 
-[Written in Python by Jeff Waugh and Scott James Remnant](https://people.gnome.org/~jdub/bzr/planet/devel/trunk/),
-Planet could be configured with a list of blog feeds and an HTML template. It would
-use these to dynamically generate a site with posts in chronological order from
-the specified feeds.
+[Được viết bằng Python bởi Jeff Waugh và Scott James Remnant](https://people.gnome.org/~jdub/bzr/planet/devel/trunk/), Planet có thể được định cấu hình với một danh sách các nguồn cấp dữ liệu blog và một mẫu HTML. Nó sẽ sử dụng những thứ này để tạo ra một trang web một cách linh hoạt với các bài đăng theo thứ tự thời gian từ các nguồn cấp dữ liệu được chỉ định.
 
-## In Search of a New Planet
+## Đi tìm một Planet mới
```

Comment:
Too bad I don't have a better way to translate this pun.

Reply:
Maybe "Đi tìm một ngôi nhà mới" work ?

Reply:
I mean if you want to trasnlate it I think mine would be a bit better

Reply:
I'm cool with that. It sounds more natural now